utils/__snapshots__/message_html_to_component.test.jsx.snap

```diff
@@ -45,10 +45,12 @@ Array [
   "
 ",
   <LatexBlock
-    content="x^2 + y^2 = z^2"
+    content="x^2 + y^2 = z^2
```

Comment:
@karlmarxlopez I think you need to revert these, then your PR should test fine.

Reply:
Yes I agree with that